nemo_skills/prompt/config/qwen/math-code-fewshot.yaml

```diff
@@ -0,0 +1,27 @@
+few_shot_examples:
+  prefix: ""
+  template: |-
+    Problem: {problem}
+    {user_end}{assistant_begin}{solution}{assistant_end}{user_begin}
```

Comment:
Why do we need user/assistant tokens here? Better to not use them and simply show examples in the first user message